### PR TITLE
Refactor Verbiage Files

### PR DESCRIPTION
**Pull request by karla-vm**
### Description

Refactoring the MCR verbiage files to match `mcpar.json` and `mlr.json` pattern.

### Related ticket(s)

N/A

---
### Notes

---
### Pre-review checklist

- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
- [ ] I have performed a self-review of my code
- [ ] I have manually tested this PR in the deployed cloud environment

---
#### Review
- [ ] Design: This work has been reviewed and approved by design, if necessary
- [ ] Product: This work has been reviewed and approved by product owner, if necessary

#### Security
_If either of the following are true, notify the team's ISSO (Information System Security Officer)._

- [ ] These changes are significant enough to require an update to the SIA.
- [ ] These changes are significant enough to require a penetration test.
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/verbiage/pages/accordion.ts [hunk 1]
```diff
@@ -4,115 +4,118 @@ export default {
...
-        {
-          type: "text",
-          as: "span",
-          content: ". <br/> <br/> <b>CMS Admin Instructions</b>",
+          type: "p",
+          children: [
+            {
+              type: "html",
+              content:
+                "<b>State User Instructions</b><br/><br/>As described at ",
```
Comment (gmrabian) [inline services/ui-src/src/verbiage/pages/accordion.ts:12]: are we able to apply these breaks with styling?

File: services/ui-src/src/verbiage/pages/accordion.ts [hunk 2]
```diff
@@ -122,40 +125,42 @@ export default {
...
+              props: {
+                href: "https://www.ecfr.gov/current/title-42/chapter-IV/subchapter-C/part-438/subpart-A/section-438.8#p-438.8(l)",
+                target: "_blank",
+                "aria-label": "42 CFR § 438.8(l) (link opens in new tab)",
+              },
+            },
+            {
+              type: "html",
+              content:
+                " a state may exclude a plan that is newly contracted with the state from this reporting for the first year of the plan's operation. These “new experience” plans must report MLRs during the next MLR reporting year in which the plan is in business with the state, even if the first year was not a full 12 months.</p>",
```
Comment (gmrabian) [inline services/ui-src/src/verbiage/pages/accordion.ts:161]: what is the close `</p>` tag for here? serving as a break?

**Review by gmrabian — APPROVED** [review]
looks good - nits about how we use these (I thought we solved the `<br>` issue at some point but maybe I'm mistaken)

File: services/ui-src/src/verbiage/pages/accordion.ts (hunk 2 above)
Comment (karla-vm) [reply services/ui-src/src/verbiage/pages/accordion.ts:161]: I'm not sure, I think I just missed it 😥 just removed it, thank you!!

File: services/ui-src/src/verbiage/pages/accordion.ts (hunk 1 above)
Comment (karla-vm) [reply services/ui-src/src/verbiage/pages/accordion.ts]: 💯 check it out!!

**Review by gmrabian — APPROVED** [review]

**Review by bangbay-bluetiger — APPROVED** [review]

**Pull request merged**